ibm/mas_devops/playbooks/cp4d/install-services-watsonstudio.yml

```diff
@@ -2,12 +2,20 @@
 - hosts: localhost
   any_errors_fatal: true
   vars:
+    # Cluster details
```

Comment:
Why do we need the CLUSTER_NAME and CLUSTER_TYPE for installing cpd services?

Reply:
Fixed. Removed CLUSTER_NAME, not really needed. Maintained CLUSTER_TYPE due ROKS limitation check i.e reload nodes.

Reply:
Actually, figured out I need CLUSTER_NAME, so the reload worker nodes process works.